Header Header
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field
string raw_data
float64 imu_velocity
float64 remaining_distance
float64 remaining_time
float64 remaining_google_time
float32 Latitude
float32 Longitude
float32 Altitude
float32 HDOP
float32 UTM_easting
float32 UTM_northing
float32 UTC
float32 gpsvelocity
float32 avgvelocity
int32 Zone
string Letter
